nodeUpdate.json

```diff
@@ -0,0 +1,21 @@
+{
+  "packageRules": [
+    {
+      "matchPackageNames": [
+        "circleci/node",
+        "node",
+        "@types/node",
+        "cimg/node"
+      ],
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
+      ],
+      "labels": [
+        "node"
```

Comment:
I'm not sure what these labels would help with. what's your intention here?

Reply:
I thought it might be useful to be able to "filter" all node related PRs from renovate. But I can remove them if you think they are overkill